### DISCUSSION TITLE
Enable Feature Gate InPlacePodVerticalScaling

### DISCUSSION THREAD

**Discussion by marvin-steinke** [k3s-io/k3s#9871] [Q&A] [answered]
Hello there, I'm trying to enable the feature gate [InPlacePodVerticalScaling](https://kubernetes.io/blog/2023/05/12/in-place-pod-resize-alpha/). I understand I should be adding this to the config file at `/etc/rancher/k3s/config.yaml` as `feature-gates=InPlacePodVerticalScaling=true`. However, I'm unsure to what component exactly I should add this arg to, is it kubelet-arg, kube-apiserver-args? Adding it to either one results in no change when checking with `kubectl api-resources` where it should be listed (I think ^^):
```
kubelet-arg:
  - feature-gates=InPlacePodVerticalScaling=true
kube-apiserver-arg: 
  - feature-gates=InPlacePodVerticalScaling=true
```

**Comment by brandond** [comment, marked as answer]
> no change when checking with kubectl api-resources

I'm not seeing that there are any new resource types associated with this featuregate, just new fields on the pod spec and status. What are you looking for, specifically?

If in doubt, set it for apiserver, kubelet, scheduler, and controller-manager.
